src/components/BrewerySearch.js

```diff
@@ -53,10 +62,10 @@ export default class BrewerySearch extends Component {
                 startDelay={1200}
                 backDelay={3000}
                 backSpeed={60}
-                loop
+                loop={true}
                 loopCount={30}
-                showCursor
-                className="typing-text"
+                showCursor={true}
+                className={"typing-text"}
               />
```

Comment:
1) `loop` is preferred over `loop={true}` as the former is the default state. It is much the same as how in `<button type="submit" disabled>Submit</button>` is the same as writing `disabled="true"`. Including the keyword defaults it to true already. 
2) `className="typing-text"` is preferred over `className={"typing-text"}`. Strings can be readily used. You would only need the blocks if you wanted className value to be conditional, as in `className={searchBy === 'city' ? 'typing-text' : 'flowing-text' }` for example.